web/src/app/schedules/CalendarEventWrapper.js

```diff
@@ -106,6 +106,7 @@ export default class CalendarEventWrapper extends Component {
               onClick={() => this.handleShowOverrideForm('replace')}
               variant='contained'
               color='primary'
+              title='Temporarily replace a user from this schedule'
```

Comment:
Since this will be locked for one specific user, can we put their name here instead of "a user"? I believe `event.title` will be the user's name.